content/admin/policy_mgmt/Exclusions.md

```diff
@@ -84,4 +107,62 @@ Alternatively, you can see a global list of existing exclusions across all appli
 
 <a href="assets/images/Application-exclusions.png" rel="lightbox" title="Manage exclusions in Policy Management"><img class="thumbnail" src="assets/images/Application-exclusions.png"/></a>
 
-
+## Protect Rule Exclusion Support Matrix
+
+All rules support URL exclusions in Protect mode.  If a rule is not listed here, it can be assumed that the rule may be excluded using URL exclusions.
```

Comment:
Do we need other teams to add their language specific Protect rules to this list?

Reply:
That'd be my assumption as well.

Reply:
why do only parts of input exclusions work on certain rules? Ex. Bean Introspection RCE Vulnerability can use Parameter but not Header? Won't this list be hard to maintain as new rules are added or existing rules change? Is there any way to generalize the description?